src/posix/platform/netif.cpp

```diff
@@ -249,6 +253,10 @@ enum
 
 static const size_t kMaxIp6Size = 1536;
 
+#ifdef OPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE
+ot::Posix::RaListener sRaListener;
```

Comment:
Should we have a option for this feature (preferably `ON` by default), so that it's possible to turn it off?